config/load_test.go

```diff
@@ -1,39 +1,38 @@
 package config
 
 import (
+	"os"
+	"strings"
 	"testing"
 
+	"github.com/spf13/viper"
 	"github.com/stretchr/testify/assert"
 )
 
-// TestIsValidEnvironment will test the method isValidEnvironment()
-func TestIsValidEnvironment(t *testing.T) {
-	t.Run("empty env", func(t *testing.T) {
-		valid := isValidEnvironment("")
-		assert.Equal(t, false, valid)
-	})
-
-	t.Run("unknown env", func(t *testing.T) {
-		valid := isValidEnvironment("unknown")
-		assert.Equal(t, false, valid)
-	})
+// TestLoadConfig will test the method Load()
+func TestLoadConfig(t *testing.T) {
+	t.Run("empty configFilePath", func(t *testing.T) {
+		// when
+		_, err := Load()
 
-	t.Run("different case of letters", func(t *testing.T) {
-		valid := isValidEnvironment("DEVELOPment")
-		assert.Equal(t, true, valid)
+		// then
+		assert.NoError(t, err)
+		assert.Equal(t, viper.GetString(ConfigFilePathKey), DefaultConfigFilePath)
 	})
 
-	t.Run("valid envs", func(t *testing.T) {
-		valid := isValidEnvironment(EnvironmentTest)
-		assert.Equal(t, true, valid)
+	t.Run("custom configFilePath overridden by ENV", func(t *testing.T) {
+		// given
+		anotherPath := "anotherPath.yml"
 
-		valid = isValidEnvironment(EnvironmentDevelopment)
-		assert.Equal(t, true, valid)
+		// when
+		os.Setenv("BUX_"+strings.ToUpper(ConfigFilePathKey), anotherPath)
```

Comment:
It is one of the most important variables. Therefore I would suggest to statically type env name. Like: 
```suggestion
		os.Setenv("BUX_CONFIG_FILE", anotherPath)
```
Why?

Because we must treat it like a part of the contract of the bux server, therefore if we change a name for variable we should also change the test, so everyone could see that we're doing something backward incompatible.

And don't forget to also change it in a `// cleanup` section (line 36)

Reply:
Makes sense, I updated this file